tvos/plugin/android/src/main/kotlin/com/tva/quickbrickplayerplugin/TVAQuickBrickPlayerView.kt

```diff
@@ -158,6 +164,9 @@ class TVAQuickBrickPlayerView(context: Context, attrs: AttributeSet?) : FrameLay
         if (params.hasKey("testVideoSrc")) {
             this.testVideoSrc = params.getString("testVideoSrc")
         }
+        if (params.hasKey("heartbeat_interval")) {
+            this.heartbeatInterval = params.getInt("heartbeat_interval")
```

Comment:
heartbeat interval is provided as string from manifest
heartbeat interval can be empty string or malformed string.

Reply:
Thank you for the feedback. Indeed. Fixed